hail/src/main/scala/is/hail/expr/ir/lowering/LoweringPass.scala

```diff
@@ -40,10 +39,10 @@ trait LoweringPass {
 }
 
 case class OptimizePass(_context: String) extends LoweringPass {
-  val context = s"optimize: ${_context}"
+  val context = s"Optimize: ${_context}"
```

Comment:
Unused?

Reply:
It's used in `LoweringPass.apply`. This should be override. Thanks for drawing my attention to it.